### DISCUSSION TITLE
How to have a color dot depending on value in a listbox ?

### DISCUSSION THREAD

**Discussion by neo7-studio-web** [directus/directus#10377] [Archive]
I saw this on Directus Project Hunt page and I'm curious to know if it's actually possible. Same thing for the separator between collections...

![color-dot2](https://user-images.githubusercontent.com/18073828/145106344-516e72cd-fcc2-43e8-b431-8c0bc379c21a.jpg)

**Comment by benhaynes** [comment]
Good eye!! 👁️   

I actually used our Directus design file to pull resources for these screenshots — and so some of the things in there were original/new designs that haven't been implemented (yet). Ths is one of those.

I'd love to have a dedicated Interface for this... or extend the standard dropdown Interface to have a color option, but to be clear, it's not in the platform as of now.

**Reply by rijkvanzanten** [reply]
For what it's worth, you can already do most of the things in the screenshot 🙂 

The status field in the screenshot is a Many to One dropdown, using the display template to render the dot + label. The navigation can already be manually ordered (and grouped, not pictured above). Dividers in the nav aren't a thing yet though

**Comment by neo7-studio-web** [comment]
Thank you guys !
